projects/1.21/assets/waystones/waystones/lang/zh_cn.json

```
@@ -0,0 +1,284 @@
+  "commands.waystones.count": "%s已激活%d个传送石碑，已拥有%d个传送石碑",
+  "commands.waystones.list.header": "传送石碑（x，y，z）由%s（拥有）：",
+  "commands.waystones.list.entry.owned": "拥有%s（%s）：%s",
+  "commands.waystones.list.entry.activated": "激活%s（%s）：%s",
+  "commands.waystones.list.footer": "拥有%d个传送石碑",
+  "commands.waystones.list.all.header": "传送石碑（x，y，z）由%s：",
+  "commands.waystones.list.all.footer": "一共有%d个传送石碑，已拥有%d个",
+  "commands.waystones.list.coordinates": "%d，%d，%d",
+  "commands.waystones.list.in_dimension": "在%s",
+  "commands.waystones.list.at_distance": "在%d个方块之外",
+  "commands.waystones.activate.success.single": "已为%2$s激活了传送石碑%1$s",
+  "commands.waystones.activate.success.multiple": "已为%2$d名玩家激活了传送石碑%1$s",
+  "commands.waystones.forget.success.single": "已为%2$s取消激活了传送石碑%1$s",
+  "commands.waystones.forget.success.multiple": "已为%2$d名玩家取消激活了传送石碑%1$s",
+  "commands.waystones.forget.all.success.single": "已为%s取消激活了所有传送石碑",
+  "commands.waystones.forget.all.success.multiple": "已为%d名玩家取消激活了所有传送石碑",
+  "commands.waystones.waystone_not_found": "当前位置并无传送石碑",
+  "config.waystones.title": "传送石碑",
+  "config.waystones.general": "通常设置",
+  "config.waystones.general.restrictedWaystones": "限制编辑列表",
+  "config.waystones.general.restrictedWaystones.tooltip": "限制他人编辑传送石碑，只允许石碑的所有者进行编辑",
+  "config.waystones.general.allowedVisibilities": "全局列表管理",
+  "config.waystones.general.allowedVisibilities.tooltip": "管理所有玩家都可见的全局石碑、石盘",
+  "config.waystones.general.warpStoneUseTime": "传送石冷却时间",
+  "config.waystones.general.warpStoneUseTime.tooltip": "使用传送石的冷却时间（单位：刻，20刻为1秒）",
+  "config.waystones.general.scrollUseTime": "传送卷轴冷却时间",
+  "config.waystones.general.scrollUseTime.tooltip": "传送卷轴的冷却时间（单位：刻，20刻为1秒）",
+  "config.waystones.general.warpPlateUseTime": "传送石盘使用时间",
+  "config.waystones.general.warpPlateUseTime.tooltip": "玩家站在传送石盘上面传送所需的时间（单位：刻，20刻为1秒）",
+  "config.waystones.teleports": "传送设置",
+  "config.waystones.teleports.enableCosts": "传送经验成本",
+  "config.waystones.teleports.enableCosts.tooltip": "设置为“关”可以无需经验，免费传送。可通过“传送要求”选项进行详细设置",
+  "config.waystones.teleports.enableCooldowns": "传送冷却时间",
+  "config.waystones.teleports.enableCooldowns.tooltip": "设置为“关”可以无冷却时间传送。可通过“传送要求”选项进行详细设置",
+  "config.waystones.teleports.warpRequirements": "传送要求",
+  "config.waystones.teleports.warpRequirements.tooltip": "传送要求列表，括号中的参数以英文逗号分隔，条件可定义为方括号中以英文逗号分隔的参数，结果按从左往右顺序应用",
+  "config.waystones.teleports.transportPets": "传送宠物",
+  "config.waystones.teleports.transportPets.tooltip": "参数：\n“ENABLED”是附近的宠物可以一同传送\n“SAME_DIMENSION”是不跨维度下附近的宠物可以一同传送\n“DISABLED”则是禁止带着宠物一同传送",
+  "config.waystones.teleports.transportLeashed": "栓绳生物传送",
+  "config.waystones.teleports.transportLeashed.tooltip": "参数：\n“ENABLED”是拴绳栓住的生物可以一同传送\n“SAME_DIMENSION”是不跨维度下拴绳栓住的生物可以一同传送\n“DISABLED”则是禁止拴绳拴住的生物进行一同传送",
+  "config.waystones.teleports.entityDenyList": "实体传送黑名单",
+  "config.waystones.teleports.entityDenyList.tooltip": "无法传送的实体列表，在传送石碑和传送石盘上生效",
+  "config.waystones.inventoryButton": "背包按钮",
```

```suggestion
  "config.waystones.inventoryButton": "物品栏界面按钮",
```
建议Inventory翻译为[物品栏](https://zh.minecraft.wiki/w/%E7%89%A9%E5%93%81%E6%A0%8F)，后面几行也需一并修改